### DISCUSSION TITLE
google.com/search-console & SiteMap

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#2719] [Q&A]
I have added my sitemap to https://search.google.com/search-console/ and using the Meta tag as verification but Google says it cant find it.  I have looked at page source and the Tag is there.  Anyone any tips or pointers

**Comment by thabaum** [comment]
@leigh-pointer 
Here are the ways Google looks at meta tags which is easy to look up I know.
https://developers.google.com/search/docs/crawling-indexing/special-tags

You probably copied and pasted it and see it in the head.  Which page has this meta tag;  all pages, default homepage or the sitemap?

So if is showing up for you, why not Google is the question?

My guess is maybe this is being rendered twice, once without, once with?  Google looks at the first one?  Just a wild idea but seems like pre-rendering methods could possibly leave out this change on the first initialization, then Oqtane adds the meta data to it a split second later after Google checks and does not see it?

Is this meta tag hard coded or dynamically loaded would be maybe a cause of the issue above.

The other thing I would do is also make sure the verification meta tag is visible on the home page and the sitemap page see if that helps.

Good luck!

**Reply by leigh-pointer** [reply]
Just went another route. Add the tag as TXT record on the DNS server! Verified!

**Reply by thabaum** [reply]
That is what I generally use if available as it seems to verify with the most success.  Thanks for the update.

**Comment by sbwalker** [comment]
@leigh-pointer what did you specify for the meta tag? I am assuming you added it to the Meta field in the Page Settings for the Home page of your site?

**Reply by leigh-pointer** [reply]
Yes! Google set the tag up. So that was copied and pasted in to the meta section on the hime page! Google couldn't verify it! Restarted app to see that would help, but that didn't.

**Reply by sbwalker** [reply]
Did you include the full meta tag:

```
<meta name="google-site-verification" content="your id" />
```

**Reply by leigh-pointer** [reply]
@sbwalker this is what Google gave me to copy and paste 
`<meta name="google-site-verification" content="wV-xmBO2Oq8jGLh4z-wRq1FionIm3u-McvAUPxGMju0" />`

And here is my Page Soucre
```
<meta charset="utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Studio-Elf Home</title>
<meta charset="UTF-8">
<meta name="google-site-verification" content="wV-xmBO2Oq8jGLh4z-wRq1FionIm3u-McvAUPxGMju0" />
<meta name="description" content="Oqtane and Studio-Elf WE'RE CURIOUS, AMBITIOUS, HUMBLE & HUNGRY.">
<meta name="keywords" content="Oqtane , Modules, Themes, Design">
<meta name="author" content="Leigh Pointer">
```

**Reply by sbwalker** [reply]
Very strange that it won't recognize the tag, as it is definitely being output during pre-render.

**Reply by sbwalker** [reply]
Any idea why most of your meta tags end in "> but the Google tag ends with />
